charts/spark-operator-chart/crds/sparkoperator.k8s.io_sparkapplications.yaml

```diff
@@ -3600,6 +3600,8 @@ spec:
                       maximum: 49151
                       minimum: 1024
                       type: integer
+                    portName:
```

Comment:
This is also needed in the `ScheduledSparkApplication` CRD.

Reply:
@liyinan926 Thanks, I updated it